openshift/openshift-copies.go

```diff
@@ -690,9 +690,11 @@ func restClientFor(config *restConfig) (*url.URL, *http.Client, error) {
 		return nil, nil, err
 	}
 
-	var httpClient *http.Client
+	httpClient := &http.Client{
+		Timeout: 1 * time.Minute,
+	}
 	if transport != http.DefaultTransport {
-		httpClient = &http.Client{Transport: transport}
+		httpClient.Transport = transport
 	}
 
```

Comment:
* Do not change `openshift-copies.go`
* In `openshift.go`: NewOpenshiftClient`, set the `Timeout` as desired.